### PR TITLE
Publish install scripts for vm agent as assets

### PR DESCRIPTION
**Pull request by reachjainrahul**
Rename and upload install-vm-agent-wrapper scripts as assets.

Signed-off-by: Rahul Jain <reachjainrahul@gmail.com>

### CODE REVIEW

**Review by Anandkumar26 — APPROVED** [review]
/LGTM

File: docs/agented-vm-guide.md [hunk 1]
```diff
@@ -212,21 +212,27 @@ documentation.
...
-installation.
+Download the [wrapper install script](../hack/install-vm-agent-wrapper.sh) from a
+[list of nephe releases](https://github.com/antrea-io/nephe/releases) and run
+the installation. For any given release <TAG> (e.g. v0.2.0), you can download
+the script as follows:
 
 ```bash
-./install-wrapper.sh --ns vm-ns --antrea-version v1.9.0 --kubeconfig ./antrea-agent.kubeconfig \
-  --antrea-kubeconfig ./antrea-agent.antrea.kubeconfig --bin ./antrea-agent
+curl https://github.com/antrea-io/nephe/releases/download/<TAG>/install-vm-agent-wrapper.sh --output install-vm-agent-wrapper.sh
```
Comment (reachjainrahul) [inline docs/agented-vm-guide.md:221]: @Anandkumar26  break this into 2. For downloading release version, do this, and for TOT do something else, like we do for nephe.yml

Comment (Anandkumar26) [reply docs/agented-vm-guide.md:221]: okay.

File: docs/agented-vm-guide.md [hunk 2]
```diff
@@ -212,21 +212,41 @@ documentation.
...
+```
+
+To download the latest version of wrapper install script, use the checked-in
+[wrapper install script](../hack/install-vm-agent-wrapper.sh)
+
+```bash
+curl https://raw.githubusercontent.com/antrea-io/nephe/main/hack/install-vm-agent-wrapper.sh --output install-vm-agent-wrapper.sh
+./install-vm-agent-wrapper.sh --ns vm-ns --antrea-version v1.9.0 --kubeconfig ./antrea-agent.kubeconfig \
```
Comment (reachjainrahul) [inline docs/agented-vm-guide.md:229]: can you separate this and mention run the install script as, it seems only for main, we are running the install.

Comment (Anandkumar26) [reply docs/agented-vm-guide.md:233]: Done

**Review by Anandkumar26 — APPROVED** [review]
/LGTM

**Pull request merged**